docs/user/site.rst

```diff
@@ -119,6 +119,10 @@ regdom \: optional
 wifi24 \: optional
     WLAN configuration for 2.4 GHz devices.
     ``channel`` must be set to a valid wireless channel for your radio.
+    ``beacon_interval`` can be specified to set a custom beacon interval in
+    time units (TU). A time unit is equivalent to 1024 microseconds.
+    If not set, the default value of 100 TU (=102.4 ms) is used.
```

Comment:
@blocktrron  sorry i'm late, but writing ms as milliseconds would have been better to differentiate from the microseconds, no?

Reply:
I'd rather replace microseconds with µs if we want to go there.

Reply:
hm, ok, both better than currently

Reply:
Fixed in 97e5434b3267ec48953d16fb92de084da116f1c9.